ui.apps/src/main/content/jcr_root/etc/google-analytics/.content.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@ShubhamGarg-AEM : pl correct the parent path .

Reply:
@Done